tests/unit/amo/helpers.js

```diff
@@ -195,6 +195,13 @@ export const fakeCategory = Object.freeze({
   weight: 1,
 });
 
+export const fakeRecommendations = Object.freeze({
+  addons: [fakeAddon, fakeAddon, fakeAddon, fakeAddon],
```

Comment:
Note: we can also write `Array(4).fill(fakeAddon)`

Reply:
Nice. :-)